man/docker-run.1.md

```diff
@@ -251,6 +252,11 @@ redirection on the host system.
 **-l**, **--label**=[]
    Set metadata on the container (e.g., --label com.example.key=value)
 
+**--isolation**="*default*"
+   Specify container's isolation level
+
+   It's quite a platform specific option which specifies isolation technology the container is going to use to isolate itself with others. For now, Linux support only 'default' (Linux Namespace), and Windows can support 'default' (Windows Server Containers) or 'hyperv' (Hyper-V Containers).
+
```

Comment:
Can we make some changes here. 

Isolation specifies the type of isolation technology used by containers. On Linux, the only supported option is 'default' (Linux Namespaces). Windows supports 'default' (the daemon default specified by `--exec-opt isolation=`), 'process' (namespace isolation only), or 'hyperv' (Hyper-V hypervisor partition based isolation). 

If no daemon `--exec-opt` for isolation is provided, the Windows daemon will default to 'process'.

The reason for this wording is that our marketing team are trying to avoid the use of "Windows Server Containers". I have been deliberately removing that from the code base. All see https://github.com/docker/docker/pull/17613 where 'process' will make far more sense.